asapdiscovery-docking/asapdiscovery/docking/scripts/run_docking_oe.py

```diff
@@ -347,10 +355,10 @@ def get_args():
         "-m",
         "--timeout",
         type=int,
-        default=30,
+        default=-1,
```

Comment:
is this intended? @hmacdope is this a good idea?

Reply:
this was from #218 and #224; I don't think the default should be to have a timer, since the expected time to completion might be different for every kind of docking run? (and, i.e. will be 10x different if you use the Omega and clash relax flags vs default) what do you think?

Reply:
I also had some issues with it being too short on my setup and will differ wildly depending on hardware. I agree its hard to judge exactly how long it should take. TBH I think changing to a large value (say 300 sec) is probably better than disabling it altogether.

Reply:
I agree that we should set an overly long timeout just to prevent jobs from hanging until eternity.

Reply:
changed to 300 seconds here: https://github.com/choderalab/asapdiscovery/pull/213/commits/aab600f4bf0e756aa1865fa0e58952fb9f95efc5